README.md

```diff
@@ -178,6 +178,11 @@ request({ method: 'POST', url: 'http://www.mocky.io/v2/5bd9984b2f00006d0006d1fd'
 ## Usage in koa
 This package supports koa server that uses [`koa-router`](https://www.npmjs.com/package/koa-router) and [`koa-bodyparser`](https://www.npmjs.com/package/koa-bodyparser)
 
+```js
+const apiMetrics = require('prometheus-api-metrics').koaMiddleware
```

Comment:
maybe it's better to destructure?

`const { koaMiddleware } = require('prometheus-api-metrics')`

Reply:
Yep, your'e right, it's shorter, changed